DeAceroDLabs/marketplace-frontend
# Fix/dmp 784 fix payment test

## What is done

[[DMP-784] fix payment test](https://deacero.atlassian.net/browse/DMP-784)

The tests were not passing because of getEstimatedDate function.

## How is done

- Deleted maxDate from tests so, it does not runs the getEstimatedDate function

## How to test

1. Run the project via `npm start`.
2. run `npm run test:coverage -- -u`. Tests should pass.
